packages/polymath-issuer/src/actions/ticker.js

```diff
@@ -97,11 +95,11 @@ export const reserve = () => async (dispatch: Function, getState: GetState) => {
   );
 };
 
-export const confirmEmail = () => async (
+export const confirmEmail = (data: Object) => async (
   dispatch: Function,
   getState: GetState
 ) => {
-  const { email } = getState().form[confirmEmailFormName].values;
+  const { email } = data;
```

Comment:
No it doesn't because there is still one that needs to be refactored (I didn't to avoid breaking something) on line 8.
From what I see `formValueSelector` is to select a single value from the form, not to retrieve all of them. And even it would be a bad pattern. The concept itself of using Redux for forms encourage this kind of pattern. I actually never encountered a situation where I needed the form data to be in the global state (that's why community build other solutions like Formik).

Reply:
Oh yeah, I understand what you mean now, I recall now reading into this code I saw this as well.
I've never had to use the form state like this before, I'll read into Formik, what you said caught my attention ;)